### PR TITLE
AN-118: Add trainer page added and added restriction for accessing pages by role

### PR DESCRIPTION
**Pull request by ArkadiuszMin**
### Jira ticket

JIRA_LINK
[AN-118](https://agh-student.atlassian.net/browse/AN-118)

### CODE REVIEW

File: frontend/sumo-championship/src/api/club.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export const getClubs = (country: string): Promise<string[]> => {
+    //TODO: add here real backend call
```
Comment (Skalakid) [inline frontend/sumo-championship/src/api/club.ts:2]: Should we wait for backend with this PR?

File: frontend/sumo-championship/src/api/trainer.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    role: Role,
+    country: string,
+    club: string,
+}
+
+export const addTrainer = async (trainer: TrainerToAdd) => {
+    console.log(trainer)
+    // const body = JSON.stringify(trainer)
+
+    // api.post("v1/trainer/add", body)()
```
Comment (Skalakid) [inline frontend/sumo-championship/src/api/trainer.ts:17]: Are we waiting for backend here?

**Review by Skalakid — COMMENTED** [review]
Please run `yarn lint` and `yarn format`. Also are your changes working? I tried to run it bu I get a lot of errors like:
```
Module not found: Error: Cannot find file: 'routes.ts' does not match the corresponding name on disk: './src/routes/ROUTES.ts'.
```

File: frontend/sumo-championship/src/api/club.ts (hunk 1 above)
Comment (ArkadiuszMin) [reply frontend/sumo-championship/src/api/club.ts:2]: ye

File: frontend/sumo-championship/src/api/trainer.ts (hunk 2 above)
Comment (ArkadiuszMin) [reply frontend/sumo-championship/src/api/trainer.ts:17]: ye

**Review by Skalakid — APPROVED** [review]

**Pull request merged**